spec/rails_helper.rb

```diff
@@ -26,3 +27,21 @@
     Role.ensure_system_roles_exist
   end
 end
+
+# Increase speed by minimizing I/O through reduced logging
+Rails.logger.level = 4
+
+if false
+# Hack by Jose Valim for all threads to share one DB connection This works on Capybara
+# because it starts the web server in a thread.
+# http://blog.plataformatec.com.br/2011/12/three-tips-to-improve-the-performance-of-your-test-suite/
+class ActiveRecord::Base
+  mattr_accessor :shared_connection
+  @@shared_connection = nil
+
+  def self.connection
+    @@shared_connection || retrieve_connection
+  end
+end
+ActiveRecord::Base.shared_connection = ActiveRecord::Base.connection
+end
```

Comment:
Yep - and this is about to be enabled, with all tests running in `:transaction` mode.